man/docker-login.1.md

```diff
@@ -18,9 +18,15 @@ Register or log in to a Docker Registry located on the specified
 do not specify a `SERVER`, the command uses Docker's public registry located at
 `https://registry-1.docker.io/` by default.  To get a username/password for Docker's public registry, create an account on Docker Hub.
 
+`docker login` requires user to use `sudo` or be `root`, except when: 
+
+1.  using docker to connect to a `docker-machine` provisioned `docker engine`.
+2.  user is added to the `docker` group.  Note this will impact the security of your system; the `docker` group is `root` equivalent.  See [Docker Daemon Attack Surface](https://docs.docker.com/articles/security/#docker-daemon-attack-surface) for details. 
+
 You can log into any public or private repository for which you have
 credentials.  When you log in, the command stores encoded credentials in
 `$HOME/.docker/config.json` on Linux or `%USERPROFILE%/.docker/config.json` on Windows.
+**Note**:  When running `sudo docker login` or `docker login` as `docker` group member, credentials are saved in `/root/.docker/config.json`.
```

Comment:
I don't think it's stored in `/root/...` if ran as a user thats in the `docker` group, only when ran using `sudo`.

Also, could you add a blank line before the note and format it in our standard "note" formatting;

```Markdown
> **Note**: ......
>
``